### PR TITLE
documentation: ZENKO-1801_remove_s3c_oob_refs_for_1.1

### PR DESCRIPTION
**Pull request by wabernat**
This PR regresses most of the changes in one of the two documentation/ZENKO-195 commits, removing OOB updates for S3C from the Zenko 1.1 documentation. 

We need it because it's bad to document products that aren't offered. 

This PR fixes #ZENKO-1801

**Special notes for your reviewers**:

Please review this as carefully as time permits. It's easier to put stuff in than to find and remove it, and there may be lingering references to this feature that I missed.

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e reset

**Comment by wabernat** [comment]
@bert-e reset

**Comment by wabernat** [comment]
@bert-e reset

File: docs/docsource/Operation-Architecture/Orbit_UI/Bucket_Management/Lifecycle_Management.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ however, the following transition rules can only be defined using API calls.
    * Setting the Days field to zero
    * Setting a custom ID
 
-See :ref: 'bucket_lifecycle_operations'
+See "Bucket Lifecycle Operationss" in the *Reference Guide*.
```
Comment (JianqinWang) [inline docs/docsource/Operation-Architecture/Orbit_UI/Bucket_Management/Lifecycle_Management.rst]: `Operations`

Comment (wabernat) [reply docs/docsource/Operation-Architecture/Orbit_UI/Bucket_Management/Lifecycle_Management.rst]: Dadblameit! I fixed that once already. 

I fixed it again.

**Comment by wabernat** [comment]
@bert-e approve

**Review by JianqinWang — APPROVED** [review]

**Review by dora-korpar — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

**Comment by ssalaues** [comment]
@bert-e bypass_build_status

**Pull request merged**